pages/search.js

```diff
@@ -40,8 +40,8 @@ export default function Search({ data }) {
   let { users, tags } = data;
   const router = useRouter();
   const { username, keyword } = router.query;
-  const [filteredUsers, setFilteredUsers] = useState([]);
   const [notFound, setNotFound] = useState();
+  const [filteredUsers, setFilteredUsers] = useState([]);
```

Comment:
not sure I understand why this line was moved?

Reply:
This was just to cluster all the `useState` together and sequenced them according to their occurrence in the code below. Did this change for better understanding as it was a bit hard for me to understand this component?

Reply:
ok, it makes it more difficult to review with so many line changes, this could have come in another PR later I think - but leave it for now